### ISSUE TITLE
SSL Cert Verification issue with global CA_Path

### ISSUE THREAD

**Issue by ffgalindo** [hep-gc/dynafed_storagestats#42]
Connection to endpoints whose SSL certificate's CA'a are not part of the grid-security certs, but rather the standard CA's fail if something like a wildcard setting:
```
locplugin.*.ca_path: /etc/grid-security/certificates/
```
is used. For instance, AWS servers will fail to verify as it only checks for CA's in the path given, not the standard ones with error:
```
Failed to establish a connection.SSL validation failed for https://... [Errno 21] Is a directory
```

**Comment by ffgalindo** [comment]
Added second connection try for WebDAV and S3 endpoints (Azure does not need it) which tries the standard CA's in case the given ca_path fails. If it fails again, then an error is raised.

This fix is available on "dev" branch as 0.10.2

**Comment by ffgalindo** [comment]
Issue resolved and merged into master.
